_includes/sidebar.njk

```diff
@@ -12,19 +12,19 @@
     <nav class="navVersions">
         <ul>
         {% for v in site.versions %}
-            <li><a href="{{ v.link|url }}" {% if version === v.slug %}class="is-current"{% endif %}>{{ v.name }}</a></li>
+            <li><a href="{{ v.link|url }}" {% if version === v.value %}class="is-current"{% endif %}>{{ v.name }}</a></li>
         {% endfor %}
         </ul>
     </nav>
 
-    <nav id="nav-main" class="navMain">
-        <ul class="navMain-sections">
-            {% if collections[version] %}
-                {% for section in collections[version] %}
-                    {% set item = baseVersion.nav[section.slug]
-                        if version == baseVersion.version and baseVersion.nav
-                        else nav[section.slug]
-                    %}
+    {# v1, v2, etc. #}
+    {% set collection_slug = ['v', version]|join %}
+
+    {% if collections[collection_slug] %}
+        <nav id="nav-main" class="navMain">
+            <ul class="navMain-sections">
+                {% for section in collections[collection_slug] %}
+                    {% set item = nav[section.slug] %}
 
                     <li class="navMain-section">
                         <h3 class="navMain-title {% if item.name === sectionTitle %}{# Marking the current section for Algolia DocSearch. #}navMain-sectionTitle{% endif %}">{{ item.name }}</h3>
```

Comment:
```suggestion
                        <h3 class="navMain-title {% if item.name === sectionTitle %}{# Marking the current section for Algolia DocSearch. #}navMain-sectionTitle{% endif %}">{{ section.title }}</h3>
```

Fixes the missing section title on archive pages.

Reply:
🙈 Just what I was looking for! Thanks a lot, @Levdbas!